Common Android/App re-signing.md

```diff
@@ -0,0 +1,54 @@
+## App re-signing
+
+In case you are provided with .apk file which is already signed, or not signed at all, you can follow these steps to re-sign it with correct signing key.
+
+You need:
+- .apk file to sign
+- .jks signing key file
+- signing password
+- Android/sdk/build-tools latest version (you get this by default when installing Android studio, but can be installed separately if needed)
+
+Let's put these in same folder.
+
+### Step 1 - check .apk file details
+If you open folder in finder, you can easily check for file size and time modified, which we'll use later to confirm app was modified by our signing key
```

Comment:
Due to the constraints of our Handbook rendering engine, it's best to separate different text formats with a blank line.

```suggestion

If you open the folder in finder, you can easily check for file size and time modified, which we'll use later to confirm the app was modified by our signing key
```